fenix/app/src/main/java/org/mozilla/fenix/compose/HorizontalPagerIndicator.kt

```diff
@@ -0,0 +1,136 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.ContentAlpha
+import androidx.compose.material.LocalContentAlpha
+import androidx.compose.material.LocalContentColor
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.platform.LocalDensity
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.IntOffset
+import androidx.compose.ui.unit.dp
+import com.google.accompanist.pager.HorizontalPager
+import com.google.accompanist.pager.PagerState
+import kotlin.math.absoluteValue
+import kotlin.math.sign
+
+/**
+ * An horizontally laid out indicator for a [HorizontalPager], that's almost the same as the
+ * accompanist HorizontalPagerIndicator, but with the ability to leave the trail of indicators
```

Comment:
I can't speak to how we could resolve the license issue. It could escalate to legal due to the nature of things. Taking a step back, it could be easier to just write a simplified more opinionated pager indicator since we don't need all the functionality and customization such as leaving a trail provided by this component. Ideally, the generic components that we do add are aligned with the design system - we would be more opinionated here because we don't need the extra customization parameters to indicate the colors, size or shape since we know all the pager indicator would just have a very specific look according to the design system.

Reply:
As discussed "offline" updated with the changes to simplify the component, Let me know your thoughts :)